luminance/src/texture.rs

```diff
@@ -781,9 +781,9 @@ pub(crate) fn opengl_target(l: Layering, d: Dim) -> GLenum {
       Dim::Cubemap => gl::TEXTURE_CUBE_MAP,
     },
     Layering::Layered => match d {
-      Dim::Dim1 => gl::TEXTURE_1D_ARRAY,
-      Dim::Dim2 => gl::TEXTURE_2D_ARRAY,
-      Dim::Dim3 => unimplemented!("3D textures array not supported"),
+      Dim::Dim1 => unimplemented!("0D textures array not supported"),
```

Comment:
I… don’t get that change. Layered 1D textures are not 0D. They are 1D textures organized in layers.